firmware/thruna/BMS/Src/App/App_Eeprom.c

```diff
@@ -1,10 +1,13 @@
 #include "App_Eeprom.h"
 #include "string.h"
 
-#define NUM_PAGES (128U)
+#define NUM_PAGES (16U)
```

Comment:
Ran into issues with address corruption above 16, could look into this further but with SD card on the way I'm not too sure how valuable that'd be

Reply:
this seems extremely sus but OK

Reply:
Oh it's definitely sus. I'll add a comment.